Dockerfile.oracle

```diff
@@ -0,0 +1,67 @@
+FROM ruby:2.2.4
+
+RUN mkdir -p /opt/homs
+RUN mkdir -p /opt/oracle
+
+COPY instantclient*.zip /opt/oracle/
+
+RUN apt-get update \
+ && apt-get install -y git \
+           libpq-dev \
```

Comment:
Sort list of arguments alphanumerically: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments. Also align it.